### ISSUE TITLE
Change the Logo in Navbar add something like Library !

### ISSUE THREAD

**Issue by prafullpandey614** [prafullpandey614/HactoberFest-2022#1]
<img width="957" alt="28 09 2022_01 41 42_REC" src="https://user-images.githubusercontent.com/81959129/192626039-c45d4802-204e-4568-8d2d-f1928faf3b52.png">
you can add a new Logo

**Comment by mohitsharma614** [comment]
Hey ! I can do this , Please this task to me

**Comment by prafullpandey614** [comment]
Welcome @mohitsharma614  I assigned you the task 
I expect a beautiful logo from you 
Note : Add logo in the Base.html file whose location is root/templates/Base.html

**Comment by iamhariom0228** [comment]
hey! I can do this  for sure, can you assign it to me?

**Comment by prafullpandey614** [comment]
Hello @iamhariom0228 Please give this repository a star and add yourself in contibuters.md
I am assigning this task to you happy Coding

**Comment by iamhariom0228** [comment]
> Hello @iamhariom0228 Please give this repository a star and add yourself in contibuters.md I am assigning this task to you happy Coding

can I directly add myself in contributors.md from your repo?

**Comment by prafullpandey614** [comment]
Follow the procedure it will help you to learn how to contribute
Yes you. An. Hange directly but it's not a good practice

**Comment by iamhariom0228** [comment]
> Follow the procedure it will help you to learn how to contribute Yes you. An. Hange directly but it's not a good practice

yeah I have created a pull request from my side and updated the contributors.md so please accept it

**Comment by prafullpandey614** [comment]
I can't see your PR please make it sure
